third_party/gpus/cuda/platform.bzl.tpl

```diff
@@ -38,6 +38,12 @@ def cudnn_library_path(version = cudnn_sdk_version()):
     else:
       return "lib64/libcudnn.so.{}".format(version)
 
+def cudnn_library_path(version = cudnn_sdk_version()):
+  print("11 version %s" % version)
```

Comment:
Done. I had removed the prints and rebased last night but wanted to wait for the results of the test first. :)